res/css/views/voip/_CallView.scss

```diff
@@ -84,6 +88,7 @@ limitations under the License.
         border-radius: 2000px;
         overflow: hidden;
         position: relative;
+        /* Blurred avatar images & pause icon for on-hold removed for now
```

Comment:
Would it make more sense to just remove it? Or you expect it to return soon...? It's always available in version control. Anyway, up to you.

Reply:
Yeah true, otherwise it'll just hang around in the css for ages